joomla/joomla-cms
# [4.3] Message after a login / logout frontend

Pull Request for Issue # .

New feature, see #30767

### Summary of Changes

There is a message after a successfully login and logout in the frontend.
### Messages
![new_feature_messages_3](https://user-images.githubusercontent.com/9271775/94781992-e8444f00-03ca-11eb-947c-fc4c1bdddfd5.png)
![new_feature_messages_2](https://user-images.githubusercontent.com/9271775/94781998-eaa6a900-03ca-11eb-9ec6-03ace2eb2a22.png)
[EDIT] Removed _successfully_ from the text in message.

### Users: Options
![new_feature_messages_4](https://user-images.githubusercontent.com/9271775/94805383-97455280-03ec-11eb-8eeb-f63ffe7fdc51.png)
[EDIT] Removed

### Testing Instructions

Backend: Go to Users > Manage > Options
(Frontend: Is not applicable.)

Set the options:
- Frontend Login Message (= Yes, default = No)
- Frontend Logout Message (= Yes, default = No)

Create a menu link:
- Menu Login form
- Menu Logout form

1) Log in and out, in case of a:
- Login form (default)
- Menu Login form
- Menu Logout form

2) Code review (developers);

- Check if the code is correct and secure.

### Actual result BEFORE applying this Pull Request

There is NOT showing a message after a suscessfully login or logout.

### Expected result AFTER applying this Pull Request

- There is showing a message after a suscessfully login. 
- There is showing a message after a suscessfully logout.

### Documentation Changes Required

Yes.
